test_prebuild/test_unit_conv/unit_conv_scheme_1.F90

```diff
@@ -31,11 +33,19 @@ subroutine unit_conv_scheme_1_run(data_array, data_array_opt, errmsg, errflg)
       ! Check values in data array
       write(error_unit,'(a,e12.4)') 'In unit_conv_scheme_1_run: checking min/max values of data array to be approximately ', target_value
       if (minval(data_array)<0.99*target_value .or. maxval(data_array)>1.01*target_value) then
-         write(errmsg,'(3(a,e12.4),a)') "Error in unit_conv_scheme_1_run, expected values of approximately ", &
+         write(errmsg,'(3(a,e12.4),a)') "Error in unit_conv_scheme_1_run, expected values for data_array of approximately ", &
                                         target_value, "  but got [ ", minval(data_array), " : ", maxval(data_array), " ]"
          errflg = 1
          return
       end if
+      ! Check values in data array2
+      write(error_unit,'(a,e12.4)') 'In unit_conv_scheme_1_run: checking min/max values of data array 2 to be approximately ', target_value2
+      if (minval(data_array2)<0.99*target_value2 .or. maxval(data_array2)>1.01*target_value2) then
```

Comment:
This probably doesn't matter, but do you know why we chose 1% deviation as the "acceptable" level of difference? That seems very large for a unit conversion.

Reply:
If the unit conversion isn't working as expected, the result will be off by a factor of 1E3 / 1E-3. Therefore, any arbitrary value that is smaller than that works.